### ISSUE TITLE
Livery selection not working properly

### ISSUE THREAD

**Issue by FranK45** [gRally/dev#341]
The livery/pattern selection drop down menus are not working, they show blank and and does not offer a selection when clicked. I can change paint color on the pattern and save succesfully.

**Comment by ghiboz** [comment]
hi @FranK45 !
all the drop are 'blank'?
![img](http://i.imgur.com/3unpn8f.png)
if you have some png's files into `Documents\gRally\Cars\131-0.1\Livery` they should be visible.. in this section there's a new drop that is not yet used

**Comment by FranK45** [comment]
Hello,

The first picture is what was shown when I opened the showroom. Second and third are when I clicked on either Liveries or Patterns boxes. The fourth picture is when I went in the Documents/grally folder an copied the liveries for the car from my backup. I then tried to removed the liveries and the issue appeared again. So the showroom seems faulty if the livery folder is empty.

Also quick note, at first when I selected the liveries I thought that they were two missing. But the drop down is scrollable, but they're are no visual indication that the menu is scrollable.

Thanks

![untitled](https://cloud.githubusercontent.com/assets/2703601/19949851/815b4f2a-a12a-11e6-9d81-96142cd28bd1.png)
![untitled2](https://cloud.githubusercontent.com/assets/2703601/19949854/8168135e-a12a-11e6-8cee-a6e7ba496b9a.png)
![untitled3](https://cloud.githubusercontent.com/assets/2703601/19949852/81653c9c-a12a-11e6-8451-2d482b85eed6.png)
![untitled4](https://cloud.githubusercontent.com/assets/2703601/19949853/8167df6a-a12a-11e6-81fb-b6a8fc498e32.png)
